### PR TITLE
Upgrade Gatsby && Blog Footer

### PR DESCRIPTION
**Pull request by escapemanuele**
- Added mdx footer to articles
- Fix use of slugify, breaking with articles with ? or . or '
- Upgraded Gatsby and other dependencies (not everything, as it was breaking the markdown)
- Resized jpg and png 
- Resized gifs (another 90MB gone)

### CODE REVIEW

File: gatsby-browser.js [hunk 1]
```diff
@@ -3,15 +3,15 @@ export { wrapPageElement } from './src'
...
-}
+//     if (
+//         query.get("product_id") === "72rJA8s-O_ZK0H7YXUOQug==" &&
+//         query.get("product_permalink") === "qdNn" &&
+//         query.has("sale_id")
+//     ) {
+//         window.localStorage.setItem("unlock_handbook", true)
+//         window.localStorage.setItem("sale_id", query.get("sale_id"))
+//     }
+// }
```
Comment (Swizec) [inline gatsby-browser.js:17]: We can fully remove this file I think

File: gatsby-config.js [hunk 2]
```diff
@@ -10,34 +14,24 @@ module.exports = {
...
+    //   resolve: "gatsby-source-filesystem",
+    //   options: {
+    //     name: "posts",
+    //     path: `${__dirname}/src/pages/blog`,
+    //   },
+    // },
+    
     "gatsby-transformer-sharp",
     "gatsby-plugin-sharp",
+    "gatsby-remark-images",
```
Comment (Swizec) [inline gatsby-config.js:27]: Oh very nice, is this new?

Comment (escapemanuele) [reply gatsby-config.js:27]: Yeah I saw that gatsby-plugin-mdx suggested this way (before and inside it)!

File: gatsby-browser.js (hunk 1 above)
Comment (escapemanuele) [reply gatsby-browser.js:17]: I left only the "export { wrapPageElement } from './src'"

**Pull request merged**